source/objects/bodyClass.m

```diff
@@ -28,48 +28,48 @@
+        dof (1,1) double {mustBeNonnegative, mustBeInteger} = 6             % (`integer`) Number of degree of freedoms (DOFs). For hydrodynamic bodies this is given in the h5 file. If not defined in the h5 file, Default = ``6``.
+        excitationIRF (1,:) double                  = []                    % (`vector`) Defines excitation Impulse Response Function, only used with the `waveClass` ``elevationImport`` type. Default = ``[]``.
+        flex (1,1) double                           = 0                     % (`integer`) Flag for flexible body, Options: 0 (off) or 1 (on). Default = ``0``.
+        gbmDOF (1,:) double {mustBeScalarOrEmpty}   = []                    % (`integer`) Number of degree of freedoms (DOFs) for generalized body mode (GBM). Default = ``[]``.
+        geometryFile (1,:) char                     = 'NONE'                % (`char array`) Path to the body geometry ``.stl`` file.
+        h5File (1,:) char                           = ''                    % (`char array`) hdf5 file containing the hydrodynamic data
+        hydroStiffness (6,6) double                 = zeros(6)              % (`6x6 float matrix`) Linear hydrostatic stiffness matrix. If the variable is nonzero, the matrix will override the h5 file values. Default = ``zeros(6)``.
+        inertia (1,:) double                        = []                    % (`1x3 float vector`) Rotational inertia or mass moment of inertia [kg*m^{2}]. Defined by the user in the following format [Ixx Iyy Izz]. Default = ``[]``.
+        initial (1,1) struct                        = struct(...            % (`structure`) Defines the initial displacement of the body.
+            'displacement',                         [0 0 0], ...            %
+            'axis',                                 [0 1 0], ...            %
+            'angle',                                0)                      % (`structure`) Defines the initial displacement of the body. ``displacement`` (`1x3 float vector`) is defined as the initial displacement of the body center of gravity (COG) [m] in the following format [x y z], Default = [``0 0 0``]. ``axis`` (`1x3 float vector`) is defined as the axis of rotation in the following format [x y z], Default = [``0 1 0``]. ``angle`` (`float`) is defined as the initial angular displacement of the body COG [rad], Default = ``0``.
+        largeXYDisplacement (1,1) struct            = struct(...            %                        
+            'option',                               0)                      %
+        linearDamping double                        = zeros(6)              % (`6x6 float matrix`) Defines linear damping coefficient matrix. Default = ``zeros(6)``.
+        mass (1,:)                                  = []                    % (`float`) Translational inertia or mass [kg]. Defined by the user or specify 'equilibrium' to set the mass equal to the fluid density times displaced volume. Default = ``[]``.
```

Comment:
@jtgrasb Another spot check, but if I update the OSWEC example with `body(1).mass = [1,1]` the simulation runs and errors out at a different point rather than in checking the input file. I realize this is difficult because this variable is either a double or a char array and cover both is challenging. Since the error was caught later this probably is a minor comment, but raises a separate question if we could convert `equilibrium` to say -1, and so we can keep mass as a (1,1) double and in the code.

Reply:
@nathanmtom Yes, covering both is challenging, but changing equilibrium to -1 would be an interesting solution. I'll also look into adding a check in the checkInputs.